arcor2_AREditor/Assets/TABLET/Scripts/GUI/OrientationJointsDetailMenu.cs

```diff
@@ -315,13 +315,13 @@ public class OrientationJointsDetailMenu : MonoBehaviour, IMenu {
         }
     }
 
-    public async void MoveHereModel(bool avoid_collision = true) {
+    public async void MoveHereModel(Vector3 position, bool avoid_collision = true) {
         List<IO.Swagger.Model.Joint> modelJoints; //joints to move the model to
         string robotId;
 
-        if (isOrientationDetail) {
+        if (true) {

```

Comment:
This can probably be removed.

Reply:
I reverted it back to original - I'm not using it in the final version.